src/fare-contracts/details/OrderDetails.tsx

```diff
@@ -61,6 +67,13 @@ export const OrderDetails = ({fareContract}: {fareContract: FareContract}) => {
         {t(FareContractTexts.details.paymentMethod)}
         {_.capitalize(fareContract?.paymentType?.join(', '))}
       </ThemeText>
+      <ThemeText
+        type="body__secondary"
+        color="secondary"
+        style={style.marginTop}
+      >
+        {t(FareContractTexts.details.totalPrice(priceString))}
```

Comment:
In case of empty prices, it is better to show an error message here? Or it is ok to ignore it?

Reply:
That's what @Sebstorvik might want to look at, but I think for the current iteration, we could write "0" if it is empty string.

Reply:
If it's an empty string, I think we should hide the text entirely. E.g. I don't think it makes sense to show anything about price for school tickets, since the student didn't purchase anything.

Reply:
@rosvik makes sense, I'll update it

Reply:
Agree. We save ourselves a bit of trouble if we our presentation is data-driven, then it isn't so fragile to future changes or unexpected data.